substrate/frame/election-provider-multi-block/src/verifier/tests.rs

```diff
@@ -911,6 +985,70 @@ mod async_verification {
 			assert!(QueuedSolution::<Runtime>::backing_iter().count().is_zero());
 		})
 	}
+
+	#[test]
+	fn malicious_miner_high_score_missing_pages() {
```

Comment:
I think this is also mostly a duplicate of the above two tests. You clear the score midway (`MockSignedNextScore::set(None);`), so this fake high score is not treated. 

I think if the above two are tested, this test is not adding much value, but please elaborate what is the difference with the above two :)